iOS/DuckDuckGo/AIChat/AIChatViewControllerManager.swift

```diff
@@ -141,3 +146,12 @@ extension AIChatViewControllerManager: RoundedPageSheetContainerViewControllerDe
         cleanUpUserContent()
     }
 }
+
+// MARK: - AIChatUserAgentHandler
+private struct AIChatUserAgentHandler: AIChatUserAgentProviding {
```

Comment:
Just a wrapper on our current UAManager to keep the AI Chat dependencies slim and focused on what it needs.